### DISCUSSION TITLE
Smart meter interface - get raw data to topic

### DISCUSSION THREAD

**Discussion by deadrabbit87** [arendst/Tasmota#19587] [Q&A]
Hi, 

i want to publish the raw data reading from the ir port of my electric meter.

How can this be done? `sensor53 d1` creates only a dump in console. 

Thanks!

**Comment by reinhard-brandstaedter** [comment]
@deadrabbit87  have you ever found out how?

**Reply by deadrabbit87** [reply]
No, unfortunately not.

But I'm still interested in...

**Reply by gemu2015** [reply]
the sml driver does not support this at all.
you may use serial bridge to monitor the serial port but this is not possible consecutive with SML.

**Reply by sfromis** [reply]
In case of using a device based on ESP32, there exists a feature in the Berry integration with Tasmota to retrieve the console output, and filter/process it. This would make it feasible to publish the interesting parts or do your own parsing directly in Berry.
